src/components/src/VRProductDisplayItem/README.md

```diff
@@ -0,0 +1,29 @@
+# VRProductDisplayItem
+
+#### Description
+
+VRProductDisplayItem is built on top of aframe and aframe-react.  Here are two links to learn more about these two libraries:
+
+https://aframe.io/
+<br>
+https://www.npmjs.com/package/aframe-react
+
+For an example of how to integrate this component into your own project checkout:
+
+`productdisplayitem.main.tsx`
+
+#### Usage
+
+```js
+import { VRProductDisplayItem } from '@elasticpath/store-components';
+```
+
+#### Example
+
+```js
+<VRProductDisplayItem handleCloseVR={() => { console.log('handlecloseVR) }} backgroundUri="https://s3.amazonaws.com/refexp/vr/1234.jpg" />
```

Comment:
The url in this example doesn't actually resolve to an image. Just FYI

Reply:
Yeah ill change that.  For some reason I thought we wouldn't want a live url? But i dont see any drawback